gulp/tasks/test.js

```diff
@@ -159,9 +159,24 @@ function testA11y() {
 }
 
 /**
- * Run the protractor browser acceptance tests.
+ * Test whether we need to load the inital data for wagtail tests.
+ * @returns {boolean} The result of the spec check
  */
-function testAcceptanceBrowser() {
+function _shouldInstallInitialData() {
+  var commandLineParams = minimist( process.argv.slice( 2 ) );
+  var specs = commandLineParams.specs;
+
+  if ( specs.length <= 0 ) {
```

Comment:
This will return `Cannot read property 'length' of undefined` if `specs` is not a command-line argument.